pyt/vulnerability_definitions/blackbox_mapping.json

```diff
@@ -8,5 +8,10 @@
         "os.path.join",
         "graham",
         "minnesota_fats"
+    ],
+    "mutates": [
```

Comment:
Just noting that this won't be used in [`vulnerabilities.py`](https://github.com/python-security/pyt/blob/master/pyt/vulnerabilities/vulnerabilities.py#L302)